types/lodash/ts3.1/common/common.d.ts

```diff
@@ -7,15 +7,15 @@ declare module "../index" {
     type Many<T> = T | ReadonlyArray<T>;
     type ImpChain<T> =
         T extends { __trapAny: any } ? Collection<any> & Function<any> & Object<any> & Primitive<any> & String :
-        T extends null | undefined ? Primitive<T> :
+        T extends null | undefined ? never :
         T extends string | null | undefined ? String :
```

Comment:
Missed this before... does the `null|undefined` on this line actually do anything?

Reply:
Yes, it prevents null and undefined from producing String, since `null extends string | null | undefined`.